jekyll/_cci2/proxy.adoc

```diff
@@ -126,7 +126,7 @@ set -a
 . /etc/environment
 ```
 +
-You will also need to follow https://docs.docker.com/network/proxy/[the Docker instructions] to make sure your containers have outbound/proxy access.
+If your containers need to use a proxy server you will need to set the followingschedulerer environment variables: `DOCKER_HTTP_PROXY`, `DOCKER_HTTPS_PROXY`, `NO_PROXY`, coresponding to those listed in https://docs.docker.com/network/proxy/[the Docker instructions]. This will ensure your containers have outbound/proxy access. For more information on creating configuration overrides, see the <<customizations#service-configuration-overrides,Customizations Guide>>.
```

Comment:
```suggestion
If your containers need to use a proxy server you will need to set the following schedulerer environment variables: `DOCKER_HTTP_PROXY`, `DOCKER_HTTPS_PROXY`, `NO_PROXY`, corresponding to those listed in https://docs.docker.com/network/proxy/[the Docker instructions]. This will ensure your containers have outbound/proxy access. For more information on creating configuration overrides, see the <<customizations#service-configuration-overrides,Customizations Guide>>.
```